build.sbt

```diff
@@ -2097,7 +2108,7 @@ lazy val `std-image` = project
       `image-polyglot-root` / "std-image.jar",
     libraryDependencies ++= Seq(
       "org.netbeans.api" % "org-openide-util-lookup" % netbeansApiVersion % "provided",
-      "org.openpnp" % "opencv" % "4.5.1-2"
+      "org.openpnp"      % "opencv"                  % "4.5.1-2"
```

Comment:
Just ran `scalafmt` on save as we are supposed to I think. I can revert these if problematic.